src/vec3.h

```diff
@@ -5,12 +5,20 @@
 #include <array>
 #include <cmath>
 
+#if defined(_MSC_VER)
+#include <immintrin.h>
+#else
+#include <x86intrin.h>
+#endif
 
-class Vec3
+class alignas(16) Vec3

```

Comment:
@Ale32 The previous definition for memory aligned class arguments was very old and not supported anymore.
This one should work for windows as well.